packaging/Makefile_distro_vars.mk

```diff
@@ -0,0 +1,108 @@
+DOT     := .
```

Comment:
This file looks like it's out of date with a packaging repo update that was landed yesterday.  Please refresh packaging/ in this repo.

Reply:
Packging repo was updated to latest commit  01542d2611d80311d67e2e6ddd9d731ca4a010f1 and this file was updated